.github/workflows/symlink_ci.yml

```diff
@@ -0,0 +1,76 @@
+name: Symlink CI
+
+on:
+  workflow_dispatch:
+  push:
+    paths:
+      - 'cmd/tools/vsymlink/**.v'
+      - '.github/workflows/symlink_ci.yml'
+  pull_request:
+    paths:
+      - 'cmd/tools/vsymlink/**.v'
+      - '.github/workflows/symlink_ci.yml'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref_name == 'master' && github.sha || github.ref_name }}
+  cancel-in-progress: true
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-20.04, macos-13]
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v4
+      - name: Build V
+        run: make -j4
+      - name: Symlink
+        run: |
+          ./v symlink
+          cd /tmp/ && v version
+          cd ~ && v version
+          echo 'println(123)' > hi.v
+          v run hi.v
+      - name: Unlink
+        run: |
+          rm $(which v)
+          v --version && exit 1 || exit 0
+      - name: Symlink (sudo)
+        run: |
+          sudo ./v symlink
+          cd /tmp/ && v version
+          cd ~ && v version
+          echo 'println(123)' > hi.v
+          v run hi.v
+      - name: Unlink
+        run: |
+          rm $(which v)
+          v --version && exit 1 || exit 0
```

Comment:
Ref. ensuring that unlinking worked worked. 
Trying to execute v --version while style linked.

![Screenshot_20240508_083530](https://github.com/vlang/v/assets/34311583/7c79addb-357f-494d-b378-031d2407696c)